src/snowflake/connector/network.py

```diff
@@ -487,6 +487,12 @@ def request(
             HTTP_HEADER_ACCEPT: accept_type,
             HTTP_HEADER_USER_AGENT: PYTHON_CONNECTOR_USER_AGENT,
         }
+        try:
+            from opentelemetry.propagate import inject
+
+            inject(headers)
+        except ModuleNotFoundError as e:
+            logger.debug(f"Opentelemtry otel injection failed because of: {e}")
```

Comment:
Not sure if the log is useful or not...

Reply:
I think it is better than put a `pass` here